src/components/schemas/expense.yaml

```diff
@@ -819,12 +819,20 @@ expense_creation_from_receipt_in:
 expense_check_policies_in:
   type: object
   additionalProperties: False
-  required:
-    - claim_amount
-    - spent_at
   properties:
+    id:
+      allOf:
+        - $ref: ./fields.yaml#/id_string
+      description: |
+        This represents the id of the expense.<br><br>
+        _Note: This field is required when you are updating an already created expense and want to check it again against the policy rules._
+      example: 'tx0agOdICdbV'
     spent_at:
-      $ref: './fields.yaml#/spent_at'
+      allOf:
+        - $ref: './fields.yaml#/spent_at'
+      description: |
+        It signifies when this expense was made. This date cannot be in future.<br><br>
+        _Note: This field is mandatory when you are creating an expense._
```

Comment:
This field is mandatory when you are creating an expense and want to check it against policy rules.